moped-editor/src/components/GridTable/Filters.js

```diff
@@ -491,6 +494,16 @@ const Filters = ({
     setIsOrToggleValue(isOr);
   };
 
+  /**
+   * Returns whether the user input value for the filterParameter is valid
+   * @param {object} filterParameter
+   * @returns {boolean}
+   */
+  const checkIsInvalidInput = (filterParameter) => {
+    // Validate input for number type fields
+    return filterParameter.type === "number" && isNaN(filterParameter.value);
+  };
```

Comment:
i left the function naming here ambiguous even though it currently only does validation for the number input type. could be extended to other types in the future (thought idk if its necessary bc cant break them as is) or i could update this to `checkIsValidNumber` or something.

Reply:
i like `checkIsValidNumber` or just `isValidNumber`. this is purely a stylistic thing, but fwiw my brain wants this function to check for "isValid" instead of "isInvalid". ie return `true` if the value is a number. 

And this function can be moved outside of the main component since it doesn't have any dependencies in the component scope 😇

Reply:
sort of related to the input type discussion further below, i noticed that whitespace characters do not break the search, which was a bit of a surprise. i was going to suggest that we trim whitespace from the user input, but that does not seem worth the trouble.

Reply:
Tagging on here to say that the regex used in the existing `removeNonIntegers` helper could be helpful here. I can still enter "1e1" because that string passes the `isNaN` test. 

[TIL that `isNaN` coerces input if needed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#description). Isn't JavaScript the best?! 😵‍💫

Reply:
Oh man :facepalm: at me not testing the input that originally caught this bug hahaha.

Thx for the catch mike, just pushed up some changes using regex instead! The way `isNaN` works is so odd :face_with_spiral_eyes: